### ISSUE TITLE
AAI broken

### ISSUE THREAD

**Issue by fbacall** [ElixirTeSS/TeSS#562]
```
F, [2018-10-30T17:38:40.168445 #17253] FATAL -- : [bd374819-7267-4c0a-980d-a8bba55a4f39]
F, [2018-10-30T17:38:40.168587 #17253] FATAL -- : [bd374819-7267-4c0a-980d-a8bba55a4f39] JSON::JWT::InvalidFormat (Invalid JSON Format):
F, [2018-10-30T17:38:40.168637 #17253] FATAL -- : [bd374819-7267-4c0a-980d-a8bba55a4f39]
F, [2018-10-30T17:38:40.168716 #17253] FATAL -- : [bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jose.rb:66:in `rescue in decode'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jose.rb:60:in `decode'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/openid_connect-0.12.0/lib/openid_connect/response_object/id_token.rb:59:in `decode'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/omniauth_openid_connect-0.1/lib/omniauth/strategies/openid_connect.rb:171:in `decode_id_token'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/omniauth_openid_connect-0.1/lib/omniauth/strategies/openid_connect.rb:160:in `access_token'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/omniauth_openid_connect-0.1/lib/omniauth/strategies/openid_connect.rb:103:in `callback_phase'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/omniauth-1.8.1/lib/omniauth/strategy.rb:236:in `callback_call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/omniauth-1.8.1/lib/omniauth/strategy.rb:188:in `call!'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/omniauth-1.8.1/lib/omniauth/strategy.rb:168:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-pjax-1.0.0/lib/rack/pjax.rb:12:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/method_override.rb:22:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:232:in `context'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:226:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/cookies.rb:670:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/remotipart-1.4.2/lib/remotipart/middleware.rb:32:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/warden-1.2.7/lib/warden/manager.rb:36:in `block in call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/warden-1.2.7/lib/warden/manager.rb:35:in `catch'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/warden-1.2.7/lib/warden/manager.rb:35:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/tempfile_reaper.rb:15:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/etag.rb:25:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/conditional_get.rb:25:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/head.rb:12:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/http/content_security_policy.rb:18:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:232:in `context'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:226:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/cookies.rb:670:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/activesupport-5.2.0/lib/active_support/callbacks.rb:98:in `run_callbacks'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/callbacks.rb:26:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/debug_exceptions.rb:61:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/railties-5.2.0/lib/rails/rack/logger.rb:38:in `call_app'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/railties-5.2.0/lib/rails/rack/logger.rb:26:in `block in call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/activesupport-5.2.0/lib/active_support/tagged_logging.rb:71:in `block in tagged'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/activesupport-5.2.0/lib/active_support/tagged_logging.rb:28:in `tagged'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/activesupport-5.2.0/lib/active_support/tagged_logging.rb:71:in `tagged'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/railties-5.2.0/lib/rails/rack/logger.rb:26:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/request_id.rb:27:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/method_override.rb:22:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/runtime.rb:22:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/activesupport-5.2.0/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/actionpack-5.2.0/lib/action_dispatch/middleware/executor.rb:14:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-2.0.5/lib/rack/sendfile.rb:111:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/rack-cors-1.0.2/lib/rack/cors.rb:97:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] vendor/bundle/ruby/2.2.0/gems/railties-5.2.0/lib/rails/engine.rb:524:in `call'
[bd374819-7267-4c0a-980d-a8bba55a4f39] /usr/lib/ruby/vendor_ruby/phusion_passenger/rack/thread_handler_extension.rb:97:in `process_request'
[bd374819-7267-4c0a-980d-a8bba55a4f39] /usr/lib/ruby/vendor_ruby/phusion_passenger/request_handler/thread_handler.rb:152:in `accept_and_process_next_request'
[bd374819-7267-4c0a-980d-a8bba55a4f39] /usr/lib/ruby/vendor_ruby/phusion_passenger/request_handler/thread_handler.rb:113:in `main_loop'
[bd374819-7267-4c0a-980d-a8bba55a4f39] /usr/lib/ruby/vendor_ruby/phusion_passenger/request_handler.rb:416:in `block (3 levels) in start_threads'
[bd374819-7267-4c0a-980d-a8bba55a4f39] /usr/lib/ruby/vendor_ruby/phusion_passenger/utils.rb:113:in `block in create_thread_and_abort_on_exception'
```

**Comment by fbacall** [comment]
```
F, [2018-11-01T11:51:19.547683 #3318] FATAL -- : [bcae03ee-d1ab-48ee-9001-7bfafaeceb43] /home/tess/.rvm/rubies/ruby-2.2.2/lib/ruby/2.2.0/base64.rb:73:in `unpack'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] /home/tess/.rvm/rubies/ruby-2.2.2/lib/ruby/2.2.0/base64.rb:73:in `strict_decode64'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] /home/tess/.rvm/rubies/ruby-2.2.2/lib/ruby/2.2.0/base64.rb:89:in `urlsafe_decode64'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jws.rb:184:in `block in decode_compact_serialized'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jws.rb:183:in `collect'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jws.rb:183:in `decode_compact_serialized'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jwt.rb:85:in `decode_compact_serialized'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/json-jwt-1.9.4/lib/json/jose.rb:63:in `decode'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/openid_connect-0.12.0/lib/openid_connect/response_object/id_token.rb:59:in `decode'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/omniauth_openid_connect-0.1/lib/omniauth/strategies/openid_connect.rb:172:in `decode_id_token'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/omniauth_openid_connect-0.1/lib/omniauth/strategies/openid_connect.rb:161:in `access_token'
[bcae03ee-d1ab-48ee-9001-7bfafaeceb43] vendor/bundle/ruby/2.2.0/gems/omniauth_openid_connect-0.1/lib/omniauth/strategies/openid_connect.rb:103:in `callback_phase'
```

**Comment by fbacall** [comment]
Fixed in 90298fa6497393918b406d11bb82ab01c89b017f
